packages/frontend/src/components/ElectionForm/Details/ElectionDetailsInlineForm.tsx

```diff
@@ -23,34 +24,34 @@ export default function ElectionDetailsInlineForm() {
 
     return (
         <Paper elevation={3}>
+        <>
             {!open &&
                 <Grid container
                     sx={{
                         m: 0,
-                        p: 1,
+                        p: 4,
                     }}
                 >
-                    <Grid xs={12}>
-                        <Typography gutterBottom variant="h4" component="h4">
-                            Election Title: {election.title}
-                        </Typography>
+                    <Grid item container xs={11}>
+                    <Grid item xs={12}>
+                                <Typography variant="h3" component="h4">
+                                    {election.title}
+                                    <ElectionStateChip state={election.state} />
```

Comment:
Nice work setting this up as a component! I have some other places in mind where this can be reused, so this makes things really convenient :D